src/views/UserList.vue

```diff
@@ -1,110 +1,204 @@
 <template>
   <div>
     <NavList />
-    <br>
-    <br>
-    <br>
-    <br>
-    <br>
-    <div>
+
+    <div class="buttons">
       <a class="Cadastro">Cadastro</a>
       <a class="Listagem">Listagem</a>
-      <br>
-      <br>
-      <br>
       <button class="btn">Cadastrar Novo Usuario</button>
     </div>
     <div class="user">
-      <br>
       <br>
       <hr>
-     <a id="cpf">CPF</a>
+      <a id="cpf">CPF</a>
       <a id="FullName">Nome Completo</a>
-      <br>  
-      <br>
       <hr>
     </div>
-    <div>
+    <div class="card text-center m-3">
       <br>
-      <div id="pagination">
-      <Pagination/>
+      <div id="usuarios">
+        <div>
+          <p id="UserCpf" v-for="item in pageOfItems"  :key="item.id">{{ item.cpf | VMask(mask)
+          }} <br></p>
+        </div>
+        <div id="nome">
+          <br>
+          <br>
+          <p id="UserName" v-for="item in pageOfItems" :key="item.id">{{ item.fullname }}  <i
+              @click="ModalOpen(item)" class="fa fa-eye" id="icon" aria-hidden="true"></i> </p>
+        </div>
+      </div>
+      <ModalEye :User_Prop="User_Prop" v-show="openModal" />
       <br>
       <br>
-      </div>
-      </div>
+      <hr>
+
+    </div>
+    <div class="card-footer pb-0 pt-3">
+      <jw-pagination :disableDefaultStyles="true" value="Continuar" :labels="customLabels" :items="users"
+        @changePage="onChangePage"></jw-pagination>
     </div>
-    
+    <br>
+  </div>
 </template>
 
 <script>
-import NavList from "../components/NavList.vue"
 import axios from 'axios'
-import Pagination from '../components/Pagination.vue'
-export default {  
+import ModalEye from "@/components/ModalEye.vue";
+import NavList from "../components/NavList.vue"
+
+const customLabels = {
+  first: '<<',
+  last: '>>',
+  previous: '<',
+  next: '>'
+};
+export default {
   components: {
     NavList,
-    Pagination
-   },
-  
+    ModalEye
+
+  },
+  data() {
+    return {
+      users: [{}],
+      User_Prop: {} ,
+      pageOfItems: [],
+      customLabels,
+      mask: "###.###.###.-##",
+      openModal: false,
+      
+    };
+  },
+  created() {
+    axios.get("/api/users/")
+      .then((res) => this.users = res.data)
+      .then((json) => {
+        this.users = json.users
```

Comment:
Por que dois .then? Já tentou dessa forma :
```suggestion
      .then((res) => this.users = res.data.users)
```

Reply:
Corrigido Kaue obrigado pela sugestão